### DISCUSSION TITLE
NAVBAR

### DISCUSSION THREAD

**Discussion by AnaGiachini** [mdbootstrap/TW-Elements-React#146] [Support from community]
Since there is no NavBar element, I have made one and I want to use a hamburger menu that is displayed only on movile screens, so I used this Tailwind property md:hidden. and it hasn't worked. I wanted to know if it is due to some configuration of TW-Elements-React

**Comment by juujisai** [comment]
Hi! TWE React should not cause issues here. The class works fine for me when I add it to a new project with some code from our footer page. Did you check whether the styles form `md:hidden` class were beeing added to the element? Maybe there is some conflict with other classes that would make the `md:hidden` class not work?
